src/block-management/block-manager.js

```diff
@@ -86,13 +91,19 @@ export default class BlockManager extends React.Component<PropsType, StateType>
 	}
 
 	onBlockTypeSelected( itemValue: string ) {
-		this.setState( { ...this.state, selectedBlockType: itemValue, blockTypePickerVisible: false } );
```

Comment:
Note: It's not necessary to return the whole state here, react will merge with the current state already